snippets/findify-head.liquid

```diff
@@ -1,5 +1,8 @@
 {% comment %} Search and Collection {% endcomment %}
 
+{% render 'findify-autocomplete-init' %}
+{% render 'findify-autocomplete-element' %}
```

Comment:
You are very confusing me, in this way autocomplete is NOT a used as a block

these 2 files should be within app side.
findify-autocomplete-element is the findify-autocomplete-block.liquid that inside have a render with findify-autocomplete-snippet that contains all the JS that IS NOT needed to be exposed to agency or cudtomization.

Take for example my findify-search-block and related snippet.